fixtures/flows/brief/rfx.js

```diff
@@ -0,0 +1,157 @@
+import * as util from '../../flows/utils'
+
+/* eslint-disable no-await-in-loop */
```

Comment:
is this still needed? if so can it be put closer to where the line is with `eslint-disable-next-line`